DQM/src/DQM/SampleValidation.cxx

```diff
@@ -117,6 +121,8 @@ namespace dqm {
     if (pdgid > 2300) label = 16; //exotic (e.g., baryon with strangeness)
```

Comment:
Have you considered what should happen if some PDGID doesn't match any of your defined criteria?

Reply:
I have considered this and plan to make changes to account for this. The current setup was put together quickly so that Jessica could proceed with validation plots. I think I will fork "exotic" into "strange baryons" and "something else" (which would then require looking at the event level to see the mystery particle). I can also expand the A' label to be any dark mediator.

Reply:
Sounds great. Just a catch-all in the end for anything unexpected can turn out to be very useful.